cms/bundles/notifications/email.py

```diff
@@ -27,11 +29,35 @@ def _send_bundle_email(bundle: Bundle, team: Team, subject: str, message: str) -
         else:
             active_user_emails.append(email)
 
+    html_template = f"bundles/notification_emails/html_variant/{email_template_name}.html"
+    plain_text_template = f"bundles/notification_emails/plain_text_variant/{email_template_name}.txt"
+    template_context = {"bundle_name": bundle.name, "bundle_inspect_url": bundle.full_inspect_url}
+
+    try:
+        plain_text_message = render_to_string(plain_text_template, template_context)
+    except TemplateDoesNotExist:  # At least the plain text template variant should always exist
+        logger.error(
+            "Plain text email template not found",
+            extra={
+                "bundle_name": bundle.name,
+                "team_name": team.name,
+                "email_subject": subject,
+                "email_template_name": email_template_name,
+            },
+        )
+        raise
+
+    try:
+        html_message = render_to_string(html_template, template_context)
+    except Exception:  # pylint: disable=broad-exception-caught
+        html_message = None
```

Comment:
Is the idea here that a plan text template must exist, but an HTML template doesn't have to?

I'm still slightly worried about this approach, because we are using a broad `Exception`. Is there a reason why we can't use `TemplateDoesNotExist`? At the moment the `template_context` is quite simple and unlikely to cause exceptions, but if things get more complicated, potentially a different exception could be raised which would go unnoticed.